ui_framework/doc_site/src/views/form/text_area_non_resizable.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  KuiTextArea,
+} from '../../../../components';
+
+
+class KuiTextAreaNonResizableExample extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value1:'Non-resizable',
+    };
```

Comment:
This can be written as:

```js
class KuiTextAreaNonResizableExample extends React.Component {
  state = {
    value1: 'Non-resizable'
  }
```

That way you don't need to do the `super(props)` stuff.

Reply:
@kjbekkelund Thanks! Nice syntax!